docs/core/diagnostics/built-in-metrics-aspnetcore.md

```diff
@@ -11,45 +11,31 @@ This article describes the metrics built-in for ASP.NET Core produced using the
-  - [Instrument: `http.server.active_requests`](#instrument-httpserveractive_requests)
-- [Meter: `Microsoft.AspNetCore.Routing`](#meter-microsoftaspnetcorerouting)
-  - [Instrument: `aspnetcore.routing.match_attempts`](#instrument-aspnetcoreroutingmatch_attempts)
-- [Meter: `Microsoft.AspNetCore.Diagnostics`](#meter-microsoftaspnetcorediagnostics)
-  - [Instrument: `aspnetcore.diagnostics.exceptions`](#instrument-aspnetcorediagnosticsexceptions)
-- [Meter: `Microsoft.AspNetCore.RateLimiting`](#meter-microsoftaspnetcoreratelimiting)
-  - [Instrument: `aspnetcore.rate_limiting.active_request_leases`](#instrument-aspnetcorerate_limitingactive_request_leases)
-  - [Instrument: `aspnetcore.rate_limiting.request_lease.duration`](#instrument-aspnetcorerate_limitingrequest_leaseduration)
-  - [Instrument: `aspnetcore.rate_limiting.queued_requests`](#instrument-aspnetcorerate_limitingqueued_requests)
-  - [Instrument: `aspnetcore.rate_limiting.request.time_in_queue`](#instrument-aspnetcorerate_limitingrequesttime_in_queue)
-  - [Instrument: `aspnetcore.rate_limiting.requests`](#instrument-aspnetcorerate_limitingrequests)
-- [Meter: `Microsoft.AspNetCore.HeaderParsing`](#meter-microsoftaspnetcoreheaderparsing)
-  - [Instrument: `aspnetcore.header_parsing.parse_errors`](#instrument-aspnetcoreheader_parsingparse_errors)
-  - [Instrument: `aspnetcore.header_parsing.cache_accesses`](#instrument-aspnetcoreheader_parsingcache_accesses)
-- [Meter: `Microsoft.AspNetCore.Server.Kestrel`](#meter-microsoftaspnetcoreserverkestrel)
-  - [Instrument: `kestrel.active_connections`](#instrument-kestrelactive_connections)
-  - [Instrument: `kestrel.connection.duration`](#instrument-kestrelconnectionduration)
-  - [Instrument: `kestrel.rejected_connections`](#instrument-kestrelrejected_connections)
-  - [Instrument: `kestrel.queued_connections`](#instrument-kestrelqueued_connections)
-  - [Instrument: `kestrel.queued_requests`](#instrument-kestrelqueued_requests)
-  - [Instrument: `kestrel.upgraded_connections`](#instrument-kestrelupgraded_connections)
-  - [Instrument: `kestrel.tls_handshake.duration`](#instrument-kestreltls_handshakeduration)
-  - [Instrument: `kestrel.active_tls_handshakes`](#instrument-kestrelactive_tls_handshakes)
-- [Meter: `Microsoft.AspNetCore.Http.Connections`](#meter-microsoftaspnetcorehttpconnections)
-  - [Instrument: `signalr.server.connection.duration`](#instrument-signalrserverconnectionduration)
-  - [Instrument: `signalr.server.active_connections`](#instrument-signalrserveractive_connections)
-
-## Meter: `Microsoft.AspNetCore.Hosting`
-
-### Instrument: `http.server.request.duration`
+## `Microsoft.AspNetCore.Hosting`
+
+The `Microsoft.AspNetCore.Hosting` metrics report high-level information about HTTP requests:
+
+- [`http.server.request.duration`](#metric-httpserverrequestduration)
+- [`http.server.active_requests`](#metric-httpserveractive_requests)
+
+### Metric: `http.server.request.duration`
```

Comment:
From a pedantic POV the 'Metric' terminology feels inaccurate, but I'm probably more pedantic and more versed in the details than the target audience. Unless we see feedback showing this 'Metric' terminology is causing confusion I'm fine with it. You are probably right that they will find the term 'Metric' more relatable even if we are abusing the terminology a bit.